support-frontend/webpack.entryPoints.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   common: {
+    '[countryGroupId]/checkout': 'pages/[countryGroupId]/checkout.tsx',
```

Comment:
I have chosen to go for file-based routing, something common in other libs and frameworks[^1][^2][^3].

This was done as I found it hard to generally follow the `Play => Webpack => tsx` route, so this should make it easier to discover. It also leaves us open to codify it later with a library. 

~I wasn't sure if we should make the `entryPoint` key `[countryGroupId]/checkout`?~ I did, again for consistency to aid discovery.

[^1]: [Astro](https://docs.astro.build/en/guides/routing/#dynamic-routes)
[^2]: [NextJs](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes)
[^3]: [Svelte](https://kit.svelte.dev/docs/routing)